tools/run_danesfield.py

```diff
@@ -224,23 +227,46 @@ def main(args):
 
     p3d_file = config['paths']['p3d_fpath']
 
+    #############################################
+    # Run VisSat pipeline
+    #############################################
+    if args.vissat:
+        aoi_config = config['paths'].get('aoi_config')
+        if aoi_config == None:
+            print("Error: Path to aoi_config file must be provided when using VisSat")
+            exit(1)
+        with open(aoi_config) as f:
+            data = json.load(f)
+
+        vissat_workdir = data['work_dir']
+        cmd_args = ["python3", "/VisSatSatelliteStereo/stereo_pipeline.py", "--config_file", aoi_config]
+        run_step(vissat_workdir, "VisSat", cmd_args)
+        cmd_args = ["python3", "/ply2txt.py", os.path.join(vissat_workdir, 'mvs_results/aggregate_3d/aggregate_3d.ply'), os.path.join(vissat_workdir, 'mvs_results/aggregate_3d/aggregate_3d.txt')]
+        run_step(vissat_workdir, "ply2txt", cmd_args)
+        cmd_args = ["/LAStools/bin/txt2las", "-i", os.path.join(vissat_workdir, 'mvs_results/aggregate_3d/aggregate_3d.txt'), "-parse", "xyz", "-o", p3d_file, "-utm", "17R", "-target_utm", "17R"]
```

Comment:
This seems problematic because we hardcode the UTM zone to "17R", so this only make sense for Florida.  We should be able to compute the zone from the lat-long coordinates.